wagtail_storages/utils.py

```diff
@@ -5,7 +5,12 @@
 
 from wagtail import VERSION as WAGTAIL_VERSION
 from wagtail.contrib.frontend_cache.utils import PurgeBatch
-from wagtail.core.models import Site
+
+if WAGTAIL_VERSION >= (3, 0):
```

Comment:
**suggestion** using a try accept block here might save one import of VERSION

Reply:
Resolved in https://github.com/torchbox/wagtail-storages/pull/28/commits/1ab46ab175ee5f01394aab82548a7d4a3f75624d